dotCMS/src/integration-test/java/com/dotmarketing/business/PermissionBitAPITest.java

```diff
@@ -0,0 +1,42 @@
+package com.dotmarketing.business;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.LicenseTestUtil;
+
+import com.dotcms.datagen.FolderDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.exception.DotDataException;
+import com.liferay.portal.model.User;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.mock;
+
+public class PermissionBitAPITest extends IntegrationTestBase  {
```

Comment:
This class needs to be added to the MainSuite in order to be run

Reply:
done https://github.com/dotCMS/core/pull/20025/commits/4f24ed8150d8e42c574b1f93351d0d0e5813d9bc#diff-8c127e4dadcd12d480a62cfc2a0dd406dbcb61bab2081608e05dfd4be0df5499R392